packages/grafana-ui/src/components/Switch/Switch.tsx

```diff
@@ -14,7 +14,7 @@ export interface Props extends Omit<HTMLProps<HTMLInputElement>, 'value'> {
 }
 
 export const Switch = React.forwardRef<HTMLInputElement, Props>(
-  ({ value, checked, disabled, onChange, id, label, ...inputProps }, ref) => {
+  ({ value, checked, onChange, id, label, disabled, readOnly, ...inputProps }, ref) => {
```

Comment:
Seems redundant to add both `disabled` and `readOnly` props here. Is this for styling purposes (i.e. a disabled field could potentially be re-enabled whereas read only might not be?)

Reply:
`disabled` is sometime not working as intended

Reply:
fixed